src/Index/MappingBuilder.php

```diff
@@ -35,6 +47,8 @@ public function buildIndexMapping(IndexConfigInterface $indexConfig)
             $mappingIndex['settings'] = $settings;
         }
 
+        $this->dispatcher->dispatch($event = new PostMappingBuildEvent($indexConfig, $mapping));
```

Comment:
Why not just `$this->dispatcher->dispatch(new PostMappingBuildEvent($indexConfig, $mapping));`?

Reply:
Fixed. https://github.com/FriendsOfSymfony/FOSElasticaBundle/pull/1734/commits/b697cc4603cbb5b42890da866d979ea1e2371f8f